File: CLASS 2/2775 부녀회장이 될테야.c
#define _CRT_SECURE_NO_WARNINGS // scanf사용시 에러무시
#include <stdio.h>

int main(void)
{
    int T;
    scanf("%d", &T);

    while (T)
    {
        int nums[14] = { 1,2,3,4,5,6,7,8,9,10,11,12,13,14 };
        int k, n;
        scanf("%d %d", &k, &n);
        int index = n - 1;

        while (k)
        {
            for (int i = 0; i < n; i++)
            {
                if (i == 0)
                    continue;
                nums[i] = nums[i - 1] + nums[i];
                index = i;
            }
            k--;
        }
        printf("%d\n", nums[index]);
        T--;
    }

    return (0);
}

File: CLASS 2/2775 부녀회장이 될테야.cs
using System;

namespace ConsoleApp2
{
    public class Program
    {
        static void Main(string[] args)
        {
            int T = int.Parse(Console.ReadLine());

            while (T > 0)
            {
                int[] nums = { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14 };

                int k = int.Parse(Console.ReadLine());
                int n = int.Parse(Console.ReadLine());
                int index = n - 1;

                while (k > 0)
                {
                    for(int i = 0; i < n; i++)
                    {
                        if (i == 0)
                            continue;
                        nums[i] = nums[i - 1] + nums[i];
                        index = i;
                    }
                    k--;
                }
                Console.WriteLine(nums[index]);
                T--;
            }
        }
    }
}
